# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: third_party/rust/gfx-backend-metal/Cargo.toml
# Commit: ef5dc3e04e5f
# Full Hash: ef5dc3e04e5f271eea0636ab3a495e95cc912f1d
# Author: Dzmitry Malyshau <dmalyshau@mozilla.com>
# Date: 2021-09-04 09:40:24
# Regressor Bug: 1726626
# File Overlap Count: 3
# Description:
#   Bug 1726626 - Move gfx/wgpu into a 3rd party dependency r=jgilbert,bholley
#   
#   This update makes wgpu a vendored dependency instead of having it in gfx/wgpu.
#   
#   ## Notes
# ==============================================================================

diff --git a/third_party/rust/gfx-backend-metal/Cargo.toml b/third_party/rust/gfx-backend-metal/Cargo.toml
deleted file mode 100644
--- a/third_party/rust/gfx-backend-metal/Cargo.toml
+++ /dev/null
@@ -1,120 +0,0 @@
-# THIS FILE IS AUTOMATICALLY GENERATED BY CARGO
-#
-# When uploading crates to the registry Cargo will automatically
-# "normalize" Cargo.toml files for maximal compatibility
-# with all versions of Cargo and also rewrite `path` dependencies
-# to registry (e.g., crates.io) dependencies
-#
-# If you believe there's an error in this file please file an
-# issue against the rust-lang/cargo repository. If you're
-# editing this file be aware that the upstream Cargo.toml
-# will likely look very different (and much more reasonable)
-
-[package]
-edition = "2018"
-name = "gfx-backend-metal"
-version = "0.9.1"
-authors = ["The Gfx-rs Developers"]
-description = "Metal API backend for gfx-rs"
-homepage = "https://github.com/gfx-rs/gfx"
-documentation = "https://docs.rs/gfx-backend-metal"
-readme = "README.md"
-keywords = ["graphics", "gamedev"]
-license = "MIT OR Apache-2.0"
-repository = "https://github.com/gfx-rs/gfx"
-[package.metadata.docs.rs]
-default-target = "x86_64-apple-darwin"
-
-[lib]
-name = "gfx_backend_metal"
-[dependencies.arrayvec]
-version = "0.5"
-
-[dependencies.auxil]
-version = "0.10"
-features = ["spirv_cross"]
-optional = true
-package = "gfx-auxil"
-
-[dependencies.bincode]
-version = "1"
-optional = true
-
-[dependencies.bitflags]
-version = "1.0"
-
-[dependencies.block]
-version = "0.1"
-
-[dependencies.cocoa-foundation]
-version = "0.1"
-
-[dependencies.copyless]
-version = "0.1.4"
-
-[dependencies.core-graphics-types]
-version = "0.1.1"
-
-[dependencies.dispatch]
-version = "0.2"
-optional = true
-
-[dependencies.foreign-types]
-version = "0.3"
-
-[dependencies.fxhash]
-version = "0.2.1"
-
-[dependencies.hal]
-version = "0.9"
-package = "gfx-hal"
-
-[dependencies.log]
-version = "0.4"
-
-[dependencies.metal]
-version = "0.23"
-features = ["private"]
-
-[dependencies.naga]
-version = "0.5"
-features = ["spv-in", "msl-out"]
-
-[dependencies.objc]
-version = "0.2.5"
-
-[dependencies.parking_lot]
-version = "0.11"
-
-[dependencies.profiling]
-version = "1"
-default-features = false
-
-[dependencies.range-alloc]
-version = "0.1"
-
-[dependencies.raw-window-handle]
-version = "0.3"
-
-[dependencies.serde]
-version = "1"
-features = ["serde_derive"]
-optional = true
-
-[dependencies.spirv_cross]
-version = "0.23"
-features = ["msl"]
-optional = true
-
-[dependencies.storage-map]
-version = "0.3"
-
-[dependencies.tempfile]
-version = "3.2"
-optional = true
-
-[features]
-cross = ["spirv_cross", "auxil", "naga/spv-out"]
-default = []
-pipeline-cache = ["tempfile", "serde", "bincode", "naga/serialize", "naga/deserialize", "naga/spv-out"]
-signpost = []